lighthouse-core/config/config.js

```diff
@@ -106,29 +106,22 @@ function cleanTrace(trace) {
   return trace;
 }
 
-function filterPasses(passes, audits, rootPath) {
+function validatePasses(passes, audits, rootPath) {
+  if (!Array.isArray(passes)) {
```

Comment:
oh, if it's null? it seems confusing to do the check in here instead of in `getGatherersNeededByAudits`